backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/versions/CassandraSchemaVersionManager.java

```diff
@@ -36,7 +36,7 @@
 
 public class CassandraSchemaVersionManager {
     public static final SchemaVersion MIN_VERSION = new SchemaVersion(12);
-    public static final SchemaVersion MAX_VERSION = new SchemaVersion(14);
+    public static final SchemaVersion MAX_VERSION = new SchemaVersion(15);
```

Comment:
I see upgrade-instruction.md missing instruction about v14/v15 update. Can you add?